app/controllers/concerns/metadata_template_actions.rb

```diff
@@ -147,4 +168,22 @@ def render_error(message)
   def set_default_sort
     @q.sorts = 'name asc' if @q.sorts.empty?
   end
+
+  def set_template_name
+    default_values = %w[none all]
+    @name = if default_values.include?(params[:metadata_template])
```

Comment:
Please update this variable to `@template_name`

Reply:
Updated in 0d2260c